pkgs/games/minetest/default.nix

```diff
@@ -1,20 +1,18 @@
-{ stdenv, fetchgit, cmake, irrlicht, libpng, bzip2, curl, libogg, jsoncpp
-, libjpeg, libXxf86vm, mesa, openal, libvorbis, xlibsWrapper, sqlite, luajit, freetype
-, gettext, doxygen
+{ stdenv, fetchurl, cmake, irrlicht, libpng, bzip2, curl, libogg, jsoncpp
+, libjpeg, libXxf86vm, mesa, openal, libvorbis, xlibsWrapper, sqlite, luajit
+, freetype, gettext, doxygen, ncurses, leveldb
 }:
 
 let
-  version = "0.4.13";
+  version = "0.4.14";
   sources = {
-    src = fetchgit {
-      url = "https://github.com/minetest/minetest.git";
-      rev = "d44fceac7e1237b00c6431ee1bb5805b602d0dcd";
-      sha256 = "034w9nv23ncdwbs4arzxfph60cfgvalh27hxprjassmz8p7ixnra";
+    src = fetchurl {
+      url = "https://github.com/minetest/minetest/archive/0.4.14.tar.gz";
```

Comment:
`fetchFromGitHub` is a better choice when using autogenerated tarballs; they tend to cause annoying checksum mismatches whenever github decides to change the compression levels or whatever.